src/credentials/examples/DeviceAttestationVerifierExample.cpp

```diff
@@ -198,14 +199,21 @@ AttestationVerificationResult ExampleDACVerifier::VerifyAttestationInformation(c
                                                  dacCertDerBuffer.data(), dacCertDerBuffer.size()) == CHIP_NO_ERROR,
                         AttestationVerificationResult::kDacSignatureInvalid);
 
-    // TODO: Re-enable this when Construct/DeconstructAttestationElements methods are introduced
-#if 0
-    ReturnErrorOnFailure(
-        DeconstructAttestationElements(attestationElements, certDeclaration, attestationNonce, timestamp, firmwareInfo));
+    ByteSpan certificationDeclarationSpan;
+    ByteSpan attestationNonceSpan;
+    uint32_t timestampDeconstructed;
+    ByteSpan firmwareInfoSpan;
+    ByteSpan vendorReservedDeconstructed[2];
+    uint16_t vendorIdDeconstructed;
+    uint16_t profileNumDeconstructed;
+    VerifyOrReturnError(DeconstructAttestationElements(attestationInfoBuffer, certificationDeclarationSpan, attestationNonceSpan,
```

Comment:
Add TODO to refactor once final vendor-specific data tags is handled.

Reply:
will update